Cognite.Simulator.Utils/ModelLibraryBase.cs

```diff
@@ -668,25 +699,6 @@ private async Task SearchAndDownloadFiles(CancellationToken token)
                 await FindModelRevisions(true, token)
                     .ConfigureAwait(false);
 
-                // Find the files that are not yet saved locally (no file path)
-                var files = _state.Values
-                    .Where(f => string.IsNullOrEmpty(f.FilePath) && f.DownloadAttempts < _config.MaxDownloadAttempts)
-                    .OrderBy(f => f.UpdatedTime)
```

Comment:
`.DownloadAttempts < _config.MaxDownloadAttempts`
this check doesn't happen anywhere anymore. is it on purpose?